### PR TITLE
Loading 16-bit tiff images

### PR DESCRIPTION
**Pull request by MyNameIsFu**
Pull Request regarding Issue #2987 

PIL.Image conversion from I;16 to L or RGB are unsuccessful as for now.
See the corresponding Issue in the Pillow GitHub (Opened 2018, so no changes to be expected)
https://github.com/python-pillow/Pillow/issues/3011

The proposed changes at least fix this issue for the mode 'I;16' and delivers a possible solution for other modes (eg. I;16B/L/N).

This results in a correct calculation of the preview thumbnail and the actual image, the annotation will be performed on.

We have used this solution on our own dataset and created annotations accordingly.

### Checklist
- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by nmanovic** [comment]
@MyNameIsFu , thanks for the contribution. Could you please add a line into CHANGELOG.md?

**Comment by MyNameIsFu** [comment]
I added a comment. Is there anything I have to change regarding the failed pipeline?

**Comment by kirill-sizov** [comment]
Hi, @MyNameIsFu, I checked your PR and met with problem: I can't create an issue with an image. This error is also the cause of the failure of the entire our testing pipeline.


I attach the image, so you can reproduce this problem.
image

**Comment by MyNameIsFu** [comment]
@kirill-sizov The attached image seems to be cropped. Can you upload it again?

**Comment by kirill-sizov** [comment]
I've uploaded the right image, this is a cropped small image that I have a problem with.

**Comment by MyNameIsFu** [comment]
Ah sorry, I'll debug it and push the changes :+1:

**Comment by MyNameIsFu** [comment]
@kirill-sizov 
I created a new Project for the test image and 2 separate tasks (from .png and from .zip). Both are created successfully and i can annotate and save correctly. The image you provided is in png format and open in from PIL.Image as mode 'rgba'. My code changes explicitly take effect on mode == 'I;16' only. Are you sure th
Screenshot from 2022-09-29 15-11-20
is is the correct image that throws the errors?

**Comment by kirill-sizov** [comment]
@MyNameIsFu, I got the following error:
Screenshot from 2022-09-29 19-07-15

And here is logs from `cvat_worker_default` container [log.zip](https://github.com/opencv/cvat/files/9676443/log.zip)

**Comment by MyNameIsFu** [comment]
Okay I looked into the Error, the CI Pipeline and the Tests and None of my changes should have any effect on the above error. I'm not even able to reproduce the Issue with CVAT. Nevertheless I tried to reproduce the Error without CVAT and instead with pure PIL and it appears to be a problem with implicit image conversion from RGBA to RGB (PNG to JPEG format).
Just as a reminder, I only touched the code regarding single-channel Images and can't think of a reason why it should have an impact on the 3 and 4 Channel images.

I'll take another look at the error and why it might differently behave on the Pipeline and my System. I also pushed Pillow version 9.2.0, just out of curiousity.

**Comment by nmanovic** [comment]
@MyNameIsFu , thanks for the contribution once again. If tests are OK, I will merge the PR. In general I hope to reimplement the solution in the future. Actually we need to have 16-bit image in UI to change contract and brightness dynamically. Even it sounds simple, the actual implementation is going to be non-trivial.

File: cvat/apps/engine/media_extractors.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import os
 import tempfile
 import shutil
+from typing import List
 import zipfile
```
Comment (nmanovic) [inline cvat/apps/engine/media_extractors.py:9]:
```suggestion
import zipfile
```

**Review by nmanovic — APPROVED** [review]

**Pull request merged**